SConstruct

```diff
@@ -324,6 +324,19 @@ if selected_platform in platform_list:
         env.vs_incs = []
         env.vs_srcs = []
 
+        from SCons import Node
+
+        def GlobRecursive(pattern, node="."):
+            results = []
+            for f in Glob(str(node) + "/*", source=True):
+                if type(f) is Node.FS.Dir:
+                    results += GlobRecursive(pattern, f)
+            results += Glob(str(node) + "/" + pattern, source=True)
+            return results
```

Comment:
I wonder whether adding `GlobRecursive` to `methods.py` could be beneficial for other developers using SCons for Godot (modules) development.

Reply:
I think the definition of `AddToVsProject` should also be moved to `methods.py`, it's (nearly) only used there:

```
drivers/SCsub
41:    env.AddToVSProject(env.drivers_sources)

SConstruct
329:        def AddToVSProject(sources):
348:        env.AddToVSProject = AddToVSProject

methods.py
561:        env.AddToVSProject(env.core_sources)
562:        env.AddToVSProject(env.main_sources)
563:        env.AddToVSProject(env.modules_sources)
564:        env.AddToVSProject(env.scene_sources)
565:        env.AddToVSProject(env.servers_sources)
566:        env.AddToVSProject(env.editor_sources
```

Reply:
Actually it shouldn't even be used in drivers/SCsub as it only caused problems worked around in 4d5a87a4225782cef357b9814fd47e4a26653e34. I will move them both to methods.py